molecule/complex_configuration/tests/test_complex_configuration.py

```diff
@@ -24,11 +31,15 @@ def test_swinsapd_service(host):
 
 def test_process_swisnapd(host):
     assert host.process.get(user=SOLARWINDS, comm=SWISNAPD)
-    assert len(host.process.filter(user=SOLARWINDS, comm="snap-plugin-col")) == 2
+    # checking default collector plugins: aosystem, processes, logs
+    assert len(host.process.filter(user=SOLARWINDS, comm="snap-plugin-col")) == 3
+    # checking default publisher plugins: publisher-appoptics, publisher-processes
     assert len(host.process.filter(user=SOLARWINDS, comm="snap-plugin-pub")) == 2
 
 
-def test_sockets(host):
+def test_sockets(host, version):
+    if version.startswith("4."):
+        pytest.skip("Using unix socket from version 4.x.x")
```

Comment:
the default is /opt/SolarWinds/Snap/run/swisnapd.sock folder on linux and similar in Program Data on windows. You can check if file exists

Reply:
done, for version 4.x.x and greater `/opt/SolarWinds/Snap/run/swisnapd.sock` file is checked